### ISSUE TITLE
[Adjust Rule] SIM908 does not detect `.get()`

### ISSUE THREAD

**Issue by MetRonnie** [MartinThoma/flake8-simplify#138]
## Desired change

* **Rule(s)**: SIM908
* **Adjustment**: The rule only detects dictionary access via `[x]`, not `.get(x)`. It should detect both.

## Explanation

I noticed that if you misread the warning you might simply replace `[x]` with `.get(x)`, and that would silence the warning without actually fixing the problem it is warning you about.

## Example

This is an example where the mentioned rule currently does not work:

```python
foo = ''
if 'a' in my_dict:
    foo = my_dict.get('a')
```
